scaper/core.py

```diff
@@ -1966,16 +1964,23 @@ def _generate_audio(self,
                     soundscape_audio, event_audio_list, scale_factor = \
                         peak_normalize(soundscape_audio, event_audio_list)
 
-                    warnings.warn(
-                        'Peak normalization applied (scale factor = {})'.format(
-                            scale_factor),
-                        ScaperWarning)
+                    ref_db_drop = 20 * np.log10(scale_factor)
+
+                    if clipping and fix_clipping:
+                        warnings.warn(
+                            'Peak normalization applied to fix clipping with '
+                            'scale factor = {}. The actual ref_db of the '
+                            'generated soundscape audio will be lower by '
+                            'approximately {:.2f}dB with respect to the target '
+                            'ref_db of {})'.format(
+                                scale_factor, ref_db_drop, self.ref_db),
```

Comment:
Let's say "change by approximately " to account for positive or negative db changes.

Reply:
again, snap!